ee/api/billing.py

```diff
@@ -74,6 +75,11 @@ def patch(self, request: Request, *args: Any, **kwargs: Any) -> Response:
                         distinct_id,
                         "billing limits updated",
                         properties={**custom_limits_usd},
+                        groups={
+                            "instance": SITE_URL,
+                            "organization": str(org.pk),
+                            "project": str(self.request.user.team.uuid) if self.request.user.team else None,
```

Comment:
It looks to be inconsistently used in the code base and has some out-of-date parts like `result["customer"] = organization.customer_id`, so I wasn't sure if it was still recommended.  I can update if that's the preferred method.

Reply:
I'd probably use it for consistency and update the method if it's incorrect. We don't use the `customer` group so you could remove that, though I don't know that there's much harm in having it there in case we ever did decide to use it (in fact, some of our customers do share a stripe customer_id across orgs...)